src/Corax/Constants.cs

```diff
@@ -1,28 +1,34 @@
 using System;
+using Sparrow;
 using Sparrow.Server;
 using Voron;
 
 namespace Corax
 {
     public static class Constants
     {
-        public const string NullValue = "NULL_VALUE";
+        public const string BeforeAllKeys = "BeforeAllKeys-a8e5f221-613e-4eae-9962-2689e7c44506";
+        public const string AfterAllKeys = "AfterAllKeys-3622a0bb-1cf4-4200-b830-5e937d57ac99";
+        
+        public static readonly string NullValue = Encodings.Utf8.GetString(new []{(byte)0});
         public static readonly ReadOnlyMemory<char> NullValueCharSpan = new(Constants.NullValue.ToCharArray());
         
         
-        public const string EmptyString = "EMPTY_STRING";
+        public const string EmptyString = @"""";
         public static readonly ReadOnlyMemory<char> EmptyStringCharSpan = new(Constants.EmptyString.ToCharArray());
         
         public const string IndexMetadata = "@index_metadata";
         public const string IndexTimeFields = "@index_time_fields";
         public const string DocumentBoost = "@document_boost";
-        
-        public static readonly Slice NullValueSlice, EmptyStringSlice, IndexMetadataSlice, DocumentBoostSlice, IndexTimeFieldsSlice;
+        public const string NullValueAsString = "NULL_VALUE";
```

Comment:
Is this still used?

Reply:
We need it for `Facets` since null cannot be a key. We could use `\0` but `NULL_VALUE` in fit behaviour from Lucene and gives compatibility if somebody already expects NULLs under this term.